pages/checkout.vue

```diff
@@ -141,12 +142,14 @@ export default {
     const { load: loadShippingMethods, shippingMethods } = useShippingMethods()
     const { toggleStoreLocatorModal, toggleLoginModal } = useUiState()
     const { user, createAccountAndLogin } = useUser()
-    const { tokenizeCard, addPaymentMethodByTokenizeCard } = usePaymentMethods()
+    const { checkoutOrder, tokenizeCard, addPaymentMethodByTokenizeCard } = usePaymentMethods()
```

Comment:
we don't want to duplicate our checkout state object

Reply:
Done